src/current/v24.3/manage-logical-data-replication.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Manage Logical Data Replication
+summary: Manage the tables that are part of the logical data replication stream.
+toc: true
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+
+Logical data replication is only supported in CockroachDB {{ site.data.products.core }} clusters.
+{{site.data.alerts.end}}
+
+Once you have **logical data replication (LDR)** running, you will need to track and manage certain parts of the job:
+
+- [Conflict resolution](#conflict-resolution): As changes to a table replicate from the source to the destination cluster, there can be conflicts during some operations that the job will handle with conflict resolution. When LDR is started, the job creates a [_dead letter queue (DLQ)_](#dead-letter-queue-dlq) table with each replicating table. LDR will send any unresolved conflicts to the DLQ, which you should monitor as LDR continues to replicate changes between the source and destination clusters. 
+- [Schema changes](#schema-changes): The tables that are part of the LDR job may require schema changes, which need to be coordinated manually. There are some schema changes that are supported while LDR jobs are actively running.
+- [Jobs](#jobs-and-ldr): All CockroachDB jobs can operate on clusters running LDR [jobs]({% link {{ page.version.version }}/show-jobs.md %}). You may want to consider where you start and how you manage [backups]({% link {{ page.version.version }}/backup-and-restore-overview.md %}), [changefeeds]({% link {{ page.version.version }}/change-data-capture-overview.md %}), [row-level TTL]({% link {{ page.version.version }}/row-level-ttl.md %}), and so on when you're running LDR.
+
+## Conflict resolution
+
+Conflicts in LDR are detected when there is either:
```

Comment:
i think this conflict section would be clearer if we separated it in to a secion on KV level conflicts and sql level conflicts. the UPDATE and cross-cluster write explanations are kv level conficts, as in the LDR is writing to a key that already exists. KV conflicts never end up in the DLQ, as LWW will automatically handle conflict resolution.

In the sql conflict subsection, we can describe how a write may not succeed because it violates a sql constraint, like and FK violation.  sql conflicts may end up in the dlq.

Reply:
I've given this a shot @msbutler please review, and feel free to use the suggestions tool to suggest edits to the markdown, if easier.